src/common/utils/index.ts

```diff
@@ -83,3 +83,20 @@ export const formatDecimal = (num: number, places = 2): string =>
  */
 export const formatPercent = (num: number, places = 0): string =>
   `${formatDecimal(100 * Math.min(1, num), places)}%`;
+
+/**
+ * Returns a formatted number, rounding to keep the specified number
+ * of significant digits. Example:
+ *
+ *   formatEstimate(123456, 2)     // 120,000
+ *   formatEstimate(123456, 3)     // 123,000
+ */
+export function formatEstimate(value: number, significantDigits = 2): string {
+  if (value <= 0) {
+    return formatInteger(value);
+  }
+  const numDigits = Math.floor(Math.log10(value));
+  const precision = Math.max(0, numDigits - significantDigits + 1);
+  const rounded = _.round(value, -1 * precision);
+  return formatInteger(rounded);
```

Comment:
Did you consider something like https://stackoverflow.com/a/36369703 ?

Reply:
😱  I didn't know about `toPrecision`, this is much better, updating!